src/entry.cpp

```diff
@@ -44,6 +49,40 @@ extern "C" __declspec(dllexport) AddonDefinition* GetAddonDef()
 	return &G::AddonDef;
 }
 
+Keybind ParseArcDPSKeybind()
+{
+	const auto arcIniPath = std::filesystem::path(G::APIDefs->Paths.GetAddonDirectory("arcdps")) / "arcdps.ini";
+	G::ArcOptions.global_mod1 = GetPrivateProfileIntA("keys", "global_mod1", VK_SHIFT, arcIniPath.string().c_str());
+	G::ArcOptions.global_mod2 = GetPrivateProfileIntA("keys", "global_mod2", VK_MENU, arcIniPath.string().c_str());
```

Comment:
ArcDPS is a bit weird with these `global_mod...` settings since I think it allows only one to be bound? Not sure about that because I stick to defaults...

Reply:
By default ArcDPS uses ALT+SHIFT+T which is mod2 and mod1. Mod multi just combines them or something weird, but can be dismissed.

Reply:
Ok, in this case I'll leave this as is, keeping in mind there might be reports (warranting additional investigations) for users:
1. having only one key bound as a `global_mod...` (I guess `GetPrivateProfileIntA` will just assign the other to default, shortcut SHOULD still work, but one button input will be simulated superfluously)
2. changing their bindings inside ArcDPS after Nexus did its initialization (quick access will stop working - and will not get fixed in next app start since Nexus will grab its old binding from `.json`, instead of getting latest values in `.ini`)